.github/workflows/CI.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  es-lint:
+    name: ESLint
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install modules
+        run: npm install
+      - name: Run ESLint
+        run: npm run lint . --ext .js,.jsx,.ts,.tsx
+  build:
+    needs: [ es-lint ]
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
```

Comment:
I don't know if it's very useful to validate multiple Node version, maybe we can agree on just using the latest LTS of Node.js?

Reply:
I think we can do that in the future. I know that if we use an LTS now (that's 16.*) it will complain that we don't have any tests. So I would suggest that we change it when we have tests. (I am researching this now but I need this in main before i can start coding)